apps/wallet-mobile/src/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -64,3 +75,23 @@ const styles = StyleSheet.create({
     lineHeight: 24,
   },
 })
+
+const useStrings = () => {
+  const intl = useIntl()
+
+  return {
+    languagepickerTitle: intl.formatMessage(messages.languagepickerTitle),
+    languagepickerSearch: intl.formatMessage(messages.languagepickerSearch),
+  }
+}
+
+const messages = defineMessages({
+  languagepickerTitle: {
+    id: 'languagepicker.title',
+    defaultMessage: '!!!Language',
+  },
+  languagepickerSearch: {
+    id: 'languagepicker.search',
+    defaultMessage: '!!!Search',
+  },
```

Comment:
isn't it available on `global` ?
```suggestion
  languagePickerTitle: {
    id: 'languagePicker.title',
    defaultMessage: '!!!Language',
  },
  languagePickerSearch: {
    id: 'languagePicker.search',
    defaultMessage: '!!!Search',
  },
```